lib/manager/nuget/extract.spec.ts

```diff
@@ -113,6 +113,41 @@ describe('manager/nuget/extract', () => {
         await extractPackageFile(otherContents, otherPackageFile, config)
       ).toMatchSnapshot();
     });
+    it('extracts msbuild-sdks from global.json', async () => {
```

Comment:
```suggestion

    it('extracts msbuild-sdks from global.json', async () => {
```

readabillity

Reply:
Fixed.

Reply:
nope, empty newlines still missing. Why you don't apply my sugesstions? You can do it from github review page as batch.

Reply:
@viceice oh.. sorry, will do in the future. (((